Bottom: 2ad7b7eb230ab13fee62e7413d6a27a5e74ecca2
Top:    ad5a844c54538f6ba5fe274edd91fcc2ee399bb0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-19 23:26:37 +0000

Refresh of hid-merge-fill-thindraw-_pcb_-

---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index 0d0851d..5344500 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -467,12 +467,12 @@ common_thindraw_pcb_pv (hidGC fg_gc, hidGC bg_gc, PinType *pv, bool drawHole, bo
 }
 
 void
-common_draw_helpers_init (HID *hid)
+common_draw_helpers_init (HID_DRAW_API *dapi)
 {
-  hid->fill_pcb_polygon     = common_fill_pcb_polygon;
-  hid->thindraw_pcb_polygon = common_thindraw_pcb_polygon;
-  hid->fill_pcb_pad         = common_fill_pcb_pad;
-  hid->thindraw_pcb_pad     = common_thindraw_pcb_pad;
-  hid->fill_pcb_pv          = common_fill_pcb_pv;
-  hid->thindraw_pcb_pv      = common_thindraw_pcb_pv;
+  dapi->fill_pcb_polygon     = common_fill_pcb_polygon;
+  dapi->thindraw_pcb_polygon = common_thindraw_pcb_polygon;
+  dapi->fill_pcb_pad         = common_fill_pcb_pad;
+  dapi->thindraw_pcb_pad     = common_thindraw_pcb_pad;
+  dapi->fill_pcb_pv          = common_fill_pcb_pv;
+  dapi->thindraw_pcb_pv      = common_thindraw_pcb_pv;
 }
diff --git a/src/hid/common/draw_helpers.h b/src/hid/common/draw_helpers.h
index 8243fe4..f9bf0cb 100644
--- a/src/hid/common/draw_helpers.h
+++ b/src/hid/common/draw_helpers.h
@@ -6,4 +6,4 @@ void common_fill_pcb_pad (hidGC gc, PadType *pad, bool clear, bool mask);
 void common_thindraw_pcb_pad (hidGC gc, PadType *pad, bool clear, bool mask);
 void common_fill_pcb_pv (hidGC gc, PinType *pv, bool drawHole, bool mask);
 void common_thindraw_pcb_pv (hidGC fg_gc, hidGC bg_gc, PinType *pv, bool drawHole, bool mask);
-void common_draw_helpers_init (HID *hid);
+void common_draw_helpers_init (HID_DRAW_API *dapi);
